### PR TITLE
Add the item fields bitrate_mode, encoder_info and encoder_settings

### PR DESCRIPTION
**Pull request by jcassette**
## Description

See beetbox/mediafile#58
See #3094

## To Do

- [x] Documentation
- [x] Changelog
- [x] Tests (nothing to do)

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -51,6 +51,7 @@ Other new things:
   yes`` in your configuration to enable.
 * :doc:`/plugins/fetchart`: A new option to change cover art format. Useful for
   DAPs that do not support some image formats.
+* :doc:`/dev/library`: Add property bitrate_mode.
```
Comment (sampsyo) [inline docs/changelog.rst:54]: We generally only use these prefixes in the changelog for plugins. Also, you might want to surround the name of the new field in backticks to make it render nicely.

**Review by sampsyo — COMMENTED** [review]
Thanks!! Maybe we should hold off on merging this until the MediaFile support is released, and then we should require that version of the MediaFile library?

File: docs/changelog.rst (hunk 1 above)
Comment (jcassette) [reply docs/changelog.rst]: Okay, I have removed the prefix and added the backticks

**Comment by jcassette** [comment]
Sure, I have updated the changelog and setup.py regarding the new required version of mediafile :-)

**Comment by jcassette** [comment]
I have also added the properties encoder_info and encoder_settings since they are available.
This would fix #3094 by allowing to get information about the VBR quality.

**Comment by jcassette** [comment]
Hi @sampsyo I have updated this :-)

**Comment by sampsyo** [comment]
Looks great; thanks!!

**Comment by jcassette** [comment]
Thank you 👍

**Pull request merged**